### ISSUE TITLE
Weird "Cut off" shadow in light mode on the text insertions page

### ISSUE THREAD

**Issue by rp1231** [poochie89/SnipDo-Extensions#111]
<img width="816" alt="Screenshot 2024-02-24 123013" src="https://github.com/poochie89/SnipDo-Extensions/assets/47380919/5ff5e312-e970-4b03-ada7-6a2d57717960">
This weird cut off shadow doesn't appear on any other page.

**Comment by poochie89** [comment]
I am not sure what you mean? The shadow around the QTranslate extension? It should only show when the cursor is over that

**Comment by rp1231** [comment]
Yes it works properly, but what I mean is that it looks weird visually.
All the elements have rounded corners but the shadow has sharp corners and it is not the same height as the element itself.

**Comment by poochie89** [comment]
Ah i see. Will tweak it a bit

**Comment by rp1231** [comment]
Thanks for the fix.
The same problem exists for the column on the left hand side.

**Comment by poochie89** [comment]
You mean the navigation menu?

**Comment by rp1231** [comment]
I think so if I understood correctly.
The hamburger menu and its items also have square edges, whereas the rest of the application has rounded edges.

**Comment by rp1231** [comment]
I think what I'm calling the hamburger menu is what you're calling the navigation menu?
The column with the text insertion, insertion extensions etc.
